### PR TITLE
Unused Traitor Items: Rad-Laser Health Analyzer

### PR DESCRIPTION
**Pull request by Iamgoofball**
Buffs the fuck out of it, no one ever uses it

Removes all the stupid UI crap like seriously who is gonna fiddle with menus for 5 hours on this shit its prolly why no one uses it

### CODE REVIEW

**Comment by kevinz000** [comment]
.... I liked the UI. Also giving a spammable stunbaton that people don't realize untill its too late is stupid. It's like a weak parapen that has a delay

File: code/game/objects/items/devices/traitordevices.dm [hunk 1]
```diff
@@ -72,7 +72,7 @@ effective or pretty fucking useless.
 	materials = list(MAT_METAL=400)
 	origin_tech = "magnets=3;biotech=5;syndicate=3"
 	var/irradiate = 1
-	var/intensity = 10 // how much damage the radiation does
+	var/intensity = 69.0 // how much damage the radiation does
```
Comment (PKPenguin321) [inline code/game/objects/items/devices/traitordevices.dm]: Nice memé

**Comment by Core0verload** [comment]
High rad damage values actually don't do more damage. Goof is just a goof.

**Comment by optimumtact** [comment]
I would accept just a buff.

**Comment by KorPhaeron** [comment]
meme pr

**Pull request closed without merging**